site/en/docs/extensions/mv3/getstarted/extensions-101/index.md

```diff
@@ -88,6 +88,8 @@ tutorials do not include a popup.
 
 ## Developing your extension {: #development }
 
+{% YouTube id='kVwX3Ci1SxU' %}
```

Comment:
Could we move this to the bottom of the section, maybe with a new subheading "Debugging", that also links to the debugging page?

Reply:
Good point, movend and added a new debugging section. Over to @jpmedley to review the writing & decide where to place the section

Reply:
Note from Joe: Can we move this video below the paragraph of text?